File: Modul 02/04/PRAK204-2410817310020-M.RizkiDinar.c
#include <stdio.h>

int main()
{
    float r, h;
    float pi = 22.0/7.0;
    scanf("%f %f", &r, &h);
    
    float vol = (pi*(r*r)*h);
    float are = (2*pi*r*h)+(2*pi*r*r);
    float cir = (2*pi*r);
    
    printf("Volume = %.2f\n", vol);
    printf("Luas = %.2f\n", are);
    printf("Keliling = %.2f\n", cir);
    return 0;
}

File: Modul 02/04/PRAK204-2410817310020-M.RizkiDinar.py
a = []

while len(a) < 2:
    a.extend(list(map(int, input().split())))

r = a[0]
h = a[1]
pi = 22/7

vol = round(pi*(r*r)*h, 2)
are = round((2*pi*r*h)+2*pi*(r*r), 2)
cir = round(2*pi*r, 2)

print("Volume = {:.2f}".format(vol))
print("Luas = {:.2f}".format(are))
print("Keliling = {:.2f}".format(cir))